omnigibson/utils/usd_utils.py

```diff
@@ -947,7 +961,7 @@ def get_relative_linear_velocity(self, prim_path):
     def get_relative_angular_velocity(self, prim_path):
         orn = self.get_position_orientation(prim_path)[1]
         angvel = self.get_angular_velocity(prim_path)
-        return T.mat2euler(T.quat2mat(orn).T @ T.euler2mat(angvel))
+        return T.quat2mat(orn).T @ angvel
```

Comment:
this `T.xxx.T` syntax is very confusing lol

Reply:
haha it's aesthetic because it's symmetrical ;D

Reply:
added a comment for clarity